form/form.go

```diff
@@ -126,10 +146,13 @@ func (f *PromptingFiller) prompt(name string, attr environschema.Attr) (interfac
 	return nil, errgo.New("too many invalid inputs")
 }
 
-// Prompt prompts for the value of the given named attribute.
-// It is always acceptable for an implementation to return a string,
-// which will be coerced to the appropriate type if possible.
 type Prompter interface {
```

Comment:
// Prompter is the interface used by PromptingFiller.
// It is used to prompt the user for a sequence of form fields
// and obtain their values.
?